apps/gatsby/package.json

```diff
@@ -9,15 +9,16 @@
     "cross-env": "7.0.3"
   },
   "dependencies": {
-    "@contentful/app-sdk": "4.9.0",
+    "@contentful/app-sdk": "4.23.0",
+    "@contentful/react-apps-toolkit": "1.2.16",
```

Comment:
If we haven't used react apps toolkit here before, we need to wrap the whole app in `SDKProvider`. How does this work without that?

Reply:
Thank you for this catch!